src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Factory/EntityTypeFactory.php

```diff
@@ -134,12 +134,7 @@ private function buildOptions($environment)
     {
         $options = [];
         if ($environment === Constants::ENVIRONMENT_PRODUCTION) {
-            $options = ['validation_groups' => ['Default', 'production'],
-                'attribute_service' => $this->attributeService
-            ];
-        }
-        if ($environment === Constants::ENVIRONMENT_TEST) {
-            $options['attribute_service'] = $this->attributeService;
+            $options = ['validation_groups' => ['Default', 'production']];
```

Comment:
Carefull here. You are removing the distinction between the production and test entity validation groups here.